gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFSInputStream.java

```diff
@@ -98,6 +112,17 @@ public synchronized int read() throws IOException {
     return (b & 0xff);
   }
 
+  private void captureDuration(String suffix, Stopwatch stopwatch, int byteRead) {
```

Comment:
Please move private methods after Public methods for easier readability.

Reply:
done